corehq/README.rst

```diff
@@ -0,0 +1,57 @@
+corehq
+############################
+
+A few broad areas of functionality are stored directly in this directory.
+
+apps
+    Most functionality lives in this directory. See README in this directory for details.
+blobs
+    The blob db stores large, schema-less pieces of data. It's where XForms and multimedia are stored.
+celery_monitoring
+    Tools to monitor `Celery <https://docs.celeryproject.org/en/stable/>`_, which we use for async task processing.
+couchapps
+    Certain couch views are stored here, instead of inside the relevant django app, because storing them separately
+    gives us performance benefits.
+dbaccessors
+    Part of ``couchapps``
+ex-submodules
+    Assorted functionality that used to live in separate repositories. See README in this directory for details.
+extensions
+    Framework for extending HQ functionality is a fork-like way. Used to handle code specific to ICDS-CAS.
+fluff
+    HQ-specific calculations that work with the ``fluff`` code in ``ex-submodules``.
+form_processor
+    Code to handle receiving, processing, and storing form submissions from CommCare mobile and Web Apps.
+messaging
+    Code to manage direct-to-user messages in CammCare HQ, most often SMS but also channels like email and
+    whatsapp. Also see the ``sms`` and ``sms`` apps in ``corehq.apps``.
+motech
+    Code relating to `MOTECH <http://docs.motechproject.org/en/latest/>`_, an open source mHealth platform that HQ
+    integrates with, particularly on large solutions projects, that also integrate with other systems such as DHIS2
+    and OpenClinica.
+pillows
+    HQ-specific mappings that use the ``pillowtop`` framework in ``ex-submodules``.
+preindex
+    Code for handling preindexing, which updates CouchDB views and ElasticSearch indices.
+    Preindex is run as part of deploy and can also be run ad hoc.
+privileges.py
+    Privileges allow HQ to limit functionality based on software plan.
+project_limits
+    Framework for throttling actions like form submissions on a domain-specific basis.
+sql_accessors
+    Stores custom postgres functions.
+sql_db
+    TODO
```

Comment:
@snopoke or @calellowitz Could you replace this? The other `sql_*` descriptions are also weak.